config/config.go

```diff
@@ -30,6 +31,11 @@ var ForceTLS bool
 var SecureCookie *securecookie.SecureCookie
 var AdminToken string
 
+// Unsafe enables a number of unsafe features when set. It is principally
+// intended to be used in development. For example, it allows the caller to
+// specify the Docker networks to join.
+var Unsafe bool
```

Comment:
This is like the `--privileged` docker feature? :joy:

Reply:
Something like that, yes!

This particular change is made in https://github.com/play-with-docker/play-with-docker/pull/411